modules/travel_pay/spec/services/expenses_client_spec.rb

```diff
@@ -194,4 +179,111 @@
       client.get_expense(veis_token, btsss_token, 'other', expense_id)
     end
   end
+
+  describe '#delete_expense' do
```

Comment:
I'm impressed with these tests overall. They're clean, simple, easy to read. Nice job.